code/parse/sexp/LuaSEXP.cpp

```diff
@@ -577,6 +577,14 @@ void LuaSEXP::parseTable() {
 					item.resize(NAME_LENGTH - 1);
 				}
 
+				// Case insensitive check if the item already exists in the list
+				for (int i = 0; i < (int)thisList.list.size(); i++) {
+					if (!stricmp(item.c_str(), thisList.list[i].c_str())) {
+						error_display(0, "Enum item '%s' already exists in list %s. Skipping!\n", item.c_str(), thisList.name.c_str());
+						continue;
```

Comment:
This will continue the inner loop, not the outer loop, and thus it won't skip adding the duplicate enum item to the list.